index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<!-- Styling -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<!-- Fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+	<!-- Scripts -->
+	<script src="https://kit.fontawesome.com/7c6af540e7.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<a class="anchor" id="top"></a>
```

Comment:
To save a line of text, you can use where your content begins i.e. class "intro" could be what you use as a return to top; however, I would like to know your reasoning for it as there are things you likely considered that I haven't.

Reply:
I've added the anchor this way so that I can position the `<a>` tag to take the navbar into account. When I tried using an ID directly on each section, it would scroll to the section but the navbar would cover some of the content.